static/src/javascripts/bootstraps/atoms/storyquestions.js

```diff
@@ -23,13 +35,15 @@ Promise.all([
     new Promise(comready),
 ]).then(() => {
     init();
-    fastdom
-        .read(
-            () =>
-                document.documentElement &&
-                document.documentElement.getBoundingClientRect().height
-        )
-        .then(height => {
-            send('resize', { height });
+    updateHeight();
+
+    // Brittle but will work
+    [...document.getElementsByClassName('user__question')]
```

Comment:
I find `[...` notation is both ugly (personal taste 😛 ) and [slow](https://twitter.com/bmeurer/status/885702158724866049) unless transpiled (do we do it for all browsers?)
Of course in this case the array will 99.99% of the time contain only `1` element, and 2 elements in the other cases.

Also in this case, why 2 copies (1 with `[...` and 1 with `slice`)?